tools/configure

```diff
@@ -663,7 +663,6 @@ infodir
 docdir
 oldincludedir
 includedir
-runstatedir
```

Comment:
Where is the (source) change to `tools/configure.ac`?

Reply:
After updating `omrmakefiles/confighelpers.m4` to check for OpenXL, I reran autoconf to generate an updated version of `configure` and `tools/configure` (since, as you mentioned, `configure` is supposed to be derived from `configure.ac`, which uses `omrmakefiles/confighelpers.m4`). However, it seems that this particular change occurs even with the previous version of  `omrmakefiles/confighelpers.m4`. Perhaps the `tools/configure` hadn't been updated to reflect the most recent changes to `tools/configure.ac`?

I included those changes in this PR, since as you mentioned, the `configure` files are supposed to be derived from the `configure.ac` files (both of which use  `omrmakefiles/confighelpers.m4`). However, if that's not what we want, then please let me know and I will remove them.

Reply:
Please use a newer version of `autoconf` to avoid removing the `runstatedir` portions.